src/glim_ros/glim_ros.cpp

```diff
@@ -37,6 +37,9 @@
 
 namespace glim {
 
+static bool init_qos_profile(const std::string& name, rmw_qos_profile_t& profile);
```

Comment:
Could you put these functions in separate header and source (e.g., `ros_qos.hpp` and `ros_qos.cpp`)?